config/nuxt/plugins.js

```diff
@@ -3,4 +3,5 @@
  */
 module.exports = [
   { src: '~/plugins/install-prompt', ssr: false },
+  { src: '~/plugins/locale-urls' },
```

Comment:
I don't understand why this diff is here. This is already in master. I don't think you can do anything about this, but maybe check if you can merge master into this branch again? Same with most of the changes below.